src/supertux/console.cpp

```diff
@@ -384,7 +385,7 @@ Console::autocomplete()
   // depending on number of hits, show matches or autocomplete
   if (cmds.empty())
   {
-    m_buffer.addLines("No known command starts with \"" + prefix + "\"");
+    m_buffer.addLines(_("No known command starts with \"") + prefix + _("\""));
```

Comment:
Hidden feature? Not really. You get his text if you try to
tab-complete a non-existing command in the ingame console.
 
    > foo[tab]
    No known command starts with "foo"
    > pla[tab]
    play()
    play_sound()
    play_demo()
    play_music()
    > play